### ISSUE TITLE
Some examples/CI still using COS image_type

### ISSUE THREAD

**Issue by apeabody** [terraform-google-modules/terraform-google-kubernetes-engine#1247]
### TL;DR

Some examples/CI still using COS image_type which is not supported in GKE v1.23.  Module updated in [v19.0](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/docs/upgrading_to_v19.0.md).

### Observed behavior

Error: googleapi: Error 400: Creation of node pools using node images based on Docker container runtimes is not supported in GKE v1.23. This is to prepare for the removal of Dockershim in Kubernetes v1.24. We recommend that you migrate to image types based on Containerd (examples).

### Terraform Configuration

```hcl
CI
```

### Terraform Version

```sh
CI
```
